report_xlsx/README.rst

```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Base report xlsx
+================
+
+This module provides a basic report class to generate xlsx report.
+
+Installation
+============
+
+Make sure you have ``xlsxwriter`` Python module installed::
+
+$ pip install xlsxwriter
+
+Usage
+=====
+
+An example of XLSX report for partners:
+
+A python class ::
+
+    from openerp.addons.report_xlsx.report.report_xlsx import ReportXlsx
+
+    class partner_xlsx(ReportXlsx):
```

Comment:
a complete detail here but why not `class PartnerXlsx()` here ? However, It must not prevent merge